### DISCUSSION TITLE
Issues with running development branch (in VS) on my local machine.

### DISCUSSION THREAD

**Discussion by MarkW06071969** [MarkW06071969/Tafe202206#17] [General]
Hi all, I cloned the repository to my local hard drive and apparently did a successful merge of the 'Mortgage Calculator' into the 'development' branch. However, though the merge was successful it wouldn't run on Visual Studio on my machine. It was only one error stopping  it (apparently a 'deployment error'),  though the code appears to be all there and the build succeeded! 

**Severity	Code	Description	Project	File	Line	Suppression State
ErrorDEP6957: Failed to connect to device '127.0.0.1' using Universal Authentication. Please verify the correct remote authentication mode is specified in the project debug settings. COMException - No connection could be made because the target machine actively refused it. (Exception from HRESULT: 0x8007274D) [0x8007274D]	Calculator**			

I am going to KT's practical tomorrow to see what the problem is. If he can fix it then I will push my local 'development' branch (which now has the 'Mortgage Calculator' and 'Currency Converter' apps merged into it) up to the GitHub Repository.

Sorry about the delay.  Will keep you updated promptly.

Regards,

Mark

**Comment by StephenHillenbrand** [comment]
You need to set your build type to local machine instead of device on the task bar drop down.
I have had this problem before too.

**Comment by MarkW06071969** [comment]
Thanks! 👍

Sent from Yahoo Mail on Android 
 
  On Mon, 5 Sept 2022 at 8:27 pm, ***@***.***> wrote:   


You need to set your build type to local machine instead of device on the task bar drop down.
I have had this problem before too.
